pkg/api/serialization_test.go

```diff
@@ -128,7 +128,7 @@ func TestList(t *testing.T) {
 	roundTripSame(t, item)
 }
 
-var nonRoundTrippableTypes = util.NewStringSet("ContainerManifest", "ContainerManifestList")
+var nonRoundTrippableTypes = util.NewStringSet("ContainerManifest", "ContainerManifestList", "ServerStatus")
```

Comment:
New objects should never appear in this list. You have a bug in your conversions if you need this.